cookbooks/aws-parallelcluster-platform/kitchen.platform-config.yml

```diff
@@ -0,0 +1,18 @@
+---
+verifier:
+  name: inspec
+  inspec_tests:
+    - test
+
+suites:
+  - name: chrony
+    run_list:
+      - recipe[aws-parallelcluster-tests::setup]
+      - recipe[aws-parallelcluster-tests::test_resource]
+    verifier:
+      controls:
+        - /tag:install_chrony/
```

Comment:
question: Shouldn't this be `tag:config_chrony_service_configured` or `tag:config_chrony`

Reply:
Agreed offline this will be addressed in next PR